backend/src/main/java/com/guestlist/guestlist_artifact/GuestListController.java

```diff
@@ -18,6 +19,7 @@
 import com.guestlist.guestlist_artifact.Model.AddGuests;
 import com.guestlist.guestlist_artifact.Model.Guests;
 
+@CrossOrigin(origins = "http://localhost:3000")
```

Comment:
To allow cross origin request from react application to backend . If this code is not there we can't call this controller from react side due to 'cross origin restriction' error , google cors error